hooks/settings/_settings_default.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+
+from ._utils import get_env
+
+
+DEVELOPMENT_MODE = get_env("DEVELOPMENT_MODE", default=True)
```

Comment:
@jimrkh Why is this an ENV variable, it isn't filled anywhere :-)

Reply:
@harpener If you run your local Docker container you are able to change development mode by env variable. You do not have to build new image with new `_settings_local.py` or edit it inside the container.